packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -730,4 +732,60 @@ export class PluginLedgerConnectorBesu
     );
     return { transaction };
   }
+  public async getBesuRecord(
+    request: BesuRecordLocatorV1Request,
+  ): Promise<BesuRecordLocatorV1Response> {
+    const fnTag = `${this.className}#getBesuRecord()`;
+    this.log.info(request);
+    //////////////////////////////////////////////
+    let abi: any = undefined;
+    const resp: BesuRecordLocatorV1Response = {};
+    const trans = request.transactionHash;
+
+    if (trans != undefined) {
+      const transaction = await this.web3.eth.getTransaction(trans);
+      if (transaction.input) {
+        resp.ledgerId = this.instanceId;
```

Comment:
@AzaharaC The ledger ID is a consortium database primary key (so different from the instance ID of the plugin itself).

You want the ledgerId to be optional otherwise we'd be making the consortium plugin a hard-dependency by accident.